admin_ui/src/components/KeySearch.vue

```diff
@@ -42,18 +40,19 @@
 </template>
 
 <script lang="ts">
+import { defineComponent } from "vue"
 import { FetchIdsConfig } from "../interfaces"
 
-export default {
+export default defineComponent({
     props: {
         fieldName: String,
         tableName: String,
-        rowID: undefined,
-        readable: undefined,
+        rowID: undefined as any,
+        readable: undefined as any,
```

Comment:
`boolean | undefined` might work here

Reply:
Unfortunately `boolean | undefined` does not work